### PR TITLE
WIP: adding mail wg e-mail roster with no members

### PR DESCRIPTION
**Pull request by spacekookie**
This is a work in progess state of the mail wg.  There's an announcement
post on the internals forum: https://internals.rust-lang.org/t/starting-a-rust-mail-wg/11678

### CODE REVIEW

**Comment by spacekookie** [comment]
Huh, I didn't think this would be okay for the CI :sweat_smile:

**Comment by dignifiedquire** [comment]
could you add me please? me@dignifiedquire.com

**Comment by hpk42** [comment]
please also add me holger at merlinux eu . thanks!

**Comment by Stupremee** [comment]
I would like to join. 
justus.k@pm.me

**Comment by mmoya** [comment]
I'd like to join mmoya [at] mmoya.org. Thanks.

**Comment by francis-du** [comment]
francis@francis.run +1

File: teams/wg-mail.toml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name = "wg-mail"
```
Comment (camelid) [inline teams/wg-mail.toml:1]: Might `wg-email` (and "Email working group") be a more descriptive and clearer name?

**Comment by spacekookie** [comment]
Usually e-mail libraries and utilities are use the term "mail", because
the "e" is implied.

**Pull request closed without merging**